.github/ISSUE_TEMPLATE/canonical_json.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Issue with canonical_json
+about: Bug, feature request or question regarding 'package:canonical_json'
+title: ''
+labels: pkg:canonical_json, pending-triage
+assignees:
+---
+ * Explain the issue,
```

Comment:
Would you not want to guide the user a bit more?

What versions of everything where you using?
What did you expect?
What did you see?

Reply:
Not for now, people tend to follow templates too much.. I want to see where this is going first :)

Mostly, I want issues to be labelled by package, and then I don't want more templates than that.